pkgs/development/tools/build-managers/buildbot/plugins.nix

```diff
@@ -1,85 +1,82 @@
 { stdenv, pythonPackages }:
 
-let
-  buildbot-pkg = pythonPackages.buildPythonPackage rec {
+# NOTE: buildbot plugins available as wheels
```

Comment:
For the main release there is now a tar.gz. I didn't check for the plugins; that might be different.